packages/web-app-files/src/components/SideBar/Details/SpaceDetails.vue

```diff
@@ -138,10 +115,64 @@ export default defineComponent({
       'currentFileOutgoingLinks'
     ]),
     ...mapGetters(['user', 'getToken']),
-
     space() {
       return this.displayedItem.value
     },
+    hasShares() {
+      return this.hasMemberShares || this.hasLinkShares
+    },
+    shareLabel() {
+      if (this.hasMemberShares && !this.hasLinkShares) {
+        return this.memberShareLabel
+      }
+      if (!this.hasMemberShares && this.hasLinkShares) {
+        return this.linkShareLabel
+      }
+      if (this.memberShareCount === 2 && this.linkShareCount === 1) {
+        return this.$gettextInterpolate(
+          'This space has %{memberShareCount} member and %{linkShareCount} link.',
+          {
+            memberShareCount: this.memberShareCount - 1,
+            linkShareCount: this.linkShareCount
+          }
+        )
+      }
+      if (this.memberShareCount === 2 && this.linkShareCount > 1) {
+        return this.$gettextInterpolate(
+          'This space has %{memberShareCount} member and %{linkShareCount} links.',
+          {
+            memberShareCount: this.memberShareCount - 1,
+            linkShareCount: this.linkShareCount
+          }
+        )
+      }
+      if (this.memberShareCount > 2 && this.linkShareCount === 1) {
+        return this.$gettextInterpolate(
+          'This space has %{memberShareCount} members and %{linkShareCount} link.',
+          {
+            memberShareCount: this.memberShareCount,
+            linkShareCount: this.linkShareCount
+          }
+        )
+      } else {
+        return this.$gettextInterpolate(
+          'This space has %{memberShareCount} members and %{linkShareCount} links.',
+          {
+            memberShareCount: this.memberShareCount,
+            linkShareCount: this.linkShareCount
+          }
+        )
+      }
+    },
```

Comment:
I still think this can be simplified to something like:

```
if (this.linkShareCount === 1) {
   return this.$gettextInterpolate('This space has %{memberShareCount} members and %{linkShareCount} link.', ....)
} else {
   return this.$gettextInterpolate('This space has %{memberShareCount} members and %{linkShareCount} links.', ....) 
}
```

Reply:
I hope I could simplify the changes a bit 😰 discussed the approach with @kulmann and we decided to start counting space members _including_ the owner/manager (previously `hasMembers` started as `> 1`), which made some logic redundant.